robot.py

```diff
@@ -118,6 +119,12 @@ def createObjects(self) -> None:
             # metres between centre of front and back wheels
             self.chassis_wheel_base = 0.517
 
+            # self.vision_pos = Translation3d(0.22, 0, 0.295) old stuff
+            self.vision_pos = Translation3d(290, -195, 235)

```

Comment:
I don't think 290 metres from the centre of the robot will be within the robot, let alone the field